src/main/resources/bundles/commonConfiguration.properties

```diff
@@ -460,4 +464,6 @@ ScheduledQueueDir=/var/spool/WildbookScheduledQueue
 
 
 uploadTmpDir = /data/upload
-numIaResultsUserCanInspect=16
\ No newline at end of file
+numIaResultsUserCanInspect=16
+
+adoptionCampaignUrl = https://donorbox.org/embed/adopt-an-animal-2
```

Comment:
Is it ok if this is exposed, or should we keep it out of the repo and just on the production servers?

Reply:
Should be fine if exposed. Just a url to the form that lives on donorbox.
Pretty much the only option to handle callback from donorbox was by providing a WB-specific callback URL. I'm adding details to adoption config in the https://docs.google.com/document/d/1o1BDHzf2FV-LTShPLgqoNJBLJnQN-tfe50lHlm92qX8/edit?ts=5e62c303# asap.

So, in other words, this url will vary by wildbook instance, and that's the only reason it lives in commonConfig. Fine if exposed.